nf_core/pipeline-template/.pre-commit-config.yaml

```diff
@@ -25,3 +25,13 @@ repos:
               subworkflows/nf-core/.*|
               .*\.snap$
           )$
+      - id: check-added-large-files
+        args: [--maxkb=5000]
+        exclude: |
+          (?x)^(
+              .*ro-crate-metadata.json$|
+              .*\.snap$|
+              lib/nfcore_external_java_deps.jar$|
+              docs/.*\.(svg|pdf)$
+          )$
+      - id: check-merge-conflict
```

Comment:
we have already https://nf-co.re/docs/nf-core-tools/api_reference/dev/pipeline_lint_tests/merge_markers, but this is a nicer approach